<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: rgb(51, 51, 51);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>551</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {	
	color: white;
	font-size: 20px;
	background-color: rgb(153, 153, 153);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_x">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: white;
	background-color: rgb(51, 153, 51);
}
QPushButton:pressed {
	background-color: rgb(51, 204, 102) ;
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_x">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>51</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {	
	color: white;
	opacity: 100
}</string>
    </property>
    <property name="text">
     <string>X =</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_x">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {	
	color: white;
	font-size: 20px;
	background-color: rgb(153, 153, 153);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_result">
    <property name="geometry">
     <rect>
      <x>209</x>
      <y>70</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {	
	color: whiite;
	font-size: 20px;
	background-color: rgb(153, 153, 153);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(102, 102, 102);</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>250</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(102, 102, 102);</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>250</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(102, 102, 102);</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>310</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(102, 102, 102);</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(102, 102, 102);</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>310</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(102, 102, 102);color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>370</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(102, 102, 102);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>370</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(102, 102, 102);</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>370</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(102, 102, 102);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_del">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>430</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(102, 102, 102);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>âŒ«</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>430</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(102, 102, 102);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>430</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(102, 102, 102);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equals">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>430</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 102, 204);</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>370</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 102, 204);</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mult">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>250</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 102, 204);</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_min">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>310</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 102, 204);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqrt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_asin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ln">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_log">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>370</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_acos">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_atan">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>310</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pow">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>430</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_open">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>190</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 102, 204);</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_close">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>190</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: rgb(51, 51, 51);</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clean">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>190</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: white;
	background-color: rgb(51, 153, 51);
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_graph">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>171</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Graph</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_credit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>130</y>
      <width>181</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Credit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mod">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
